composer.json

```diff
@@ -27,7 +27,8 @@
         "ext-curl": "*"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "dev-master"
+        "satooshi/php-coveralls": "dev-master",
+        "phpunit/phpunit": ">=4.0.0"
```

Comment:
How this is helpful knowing, that:

* Travis CI has PHPUnit installed already
* Most of developers have PHPUnit installed anyway (e.g. via Composer globally or PHAR)